### ISSUE TITLE
[Feature] Allow disabling console log output

### ISSUE THREAD

**Issue by IanVS** [storybookjs/test-runner#355]
**Describe the Feature**

After https://github.com/storybookjs/test-runner/pull/157, we get nice output of console logs, but I'm finding that sometimes the entire dom dump from testing-library is repeated twice, which makes it really hard to find the names of the actual tests that are failing.

I'd like to propose an option that disables printing of console logs, or at least find a way to avoid showing the same text that is in the error message. The error message itself is helpfully truncated, but the browser logs are not, so this can be a very long dump indeed:

For example, this is the kind of output I would like to avoid:

```
  warn: You are using Testing Library's `screen` object. Use `within(canvasElement)` instead.
    More info: https://storybook.js.org/docs/react/essentials/interactions

    error: TestingLibraryElementError: Unable to find an accessible element with the role "dialog" and name "Add authenticator"

    There are no accessible roles. But there might be some inaccessible roles. If you wish to access them, then set the `hidden` option to `true`. Learn more about this here: https://testing-library.com/docs/dom-testing-library/api-queries#byrole

    Ignored nodes: comments, script, style
    <body
      class="sb-show-main sb-main-padded"
      style=""
    >
      
      
      <div
        class="sb-preparing-story sb-wrapper"
      >
        
      
        <div
          class="sb-loader"
        />
        

      </div>
      


      <div
        class="sb-preparing-docs sb-wrapper"
      >
```

**Comment by yannbf** [comment]
Hey there! I recently merged #405 which does not disable the logs, but it does remove the duplicated error message. Thought I'd let you know, I think this will make your experience much better!

**Comment by IanVS** [comment]
Seems like a good fix, thanks.
